angr/analyses/decompiler/utils.py

```diff
@@ -725,6 +726,26 @@ def calls_in_graph(graph: networkx.DiGraph) -> int:
     return counter.calls
 
 
+def find_block_by_addr(graph: networkx.DiGraph, addr, insn_addr=False):
+    if insn_addr:
+
+        def _get_addr(b):
+            return b.statements[0].ins_addr
+
+    else:
+
+        def _get_addr(b):
+            return b.addr
```

Comment:
Defining functions differently in different branches is a bad programming practice. Please refactor it.